apm-agent-plugins/apm-redis-plugin/apm-lettuce-3-tests/src/test/java/co/elastic/apm/agent/redis/lettuce/Lettuce3VersionsIT.java

```diff
@@ -0,0 +1,57 @@
+/*-
+ * #%L
+ * Elastic APM Java agent
+ * %%
+ * Copyright (C) 2018 - 2019 Elastic and contributors
+ * %%
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ * #L%
+ */
+package co.elastic.apm.agent.redis.lettuce;
+
+import co.elastic.apm.agent.TestClassWithDependencyRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.List;
+
+@RunWith(Parameterized.class)
+public class Lettuce3VersionsIT {
+
+    private final TestClassWithDependencyRunner runner;
+
+    public Lettuce3VersionsIT(List<String> dependencies) throws Exception {
+        System.setProperty("io.lettuce.core.kqueue", "false");
```

Comment:
A native-OS specific Netty IO component. Ivy has problems resolving the Netty dependencies because Lettuce imports the Netty BOM. That's why I have to explicitly add the netty dependencies. Netty is modular and consists of many components. For the tests it's easier to just add `netty-all` but that doesn't include the kqueue module.

Reply:
OK, vaguely understood, just wanted to make sure this is test-specific